examples/hcp-ec2-demo/main.tf

```diff
@@ -78,8 +78,16 @@ module "aws_ec2_consul_client" {
   install_demo_app         = var.install_demo_app
   root_token               = hcp_consul_cluster_root_token.token.secret_id
   security_group_id        = module.aws_hcp_consul.security_group_id
+  ssh_key                  = var.ssh ? tls_private_key.ssh.private_key_pem : ""
   ssh_keyname              = var.ssh ? aws_key_pair.hcp_ec2[0].key_name : ""
```

Comment:
`ssh_key` and `ssh_keyname` are now required parameters with the provisioner changes, right? I think that is probably ok, but something to think about is how this interacts with `ssm` variable as well. Can they coexist together? I assume so.

Reply:
Yes both are now required. I made a change to remove the `default` `ssh_keyname` to reflect that in: https://github.com/hashicorp/terraform-aws-hcp-consul/pull/83/commits/5e1254beb95b858928c508c84542a3af0e0fb205

They can coexist, I just ran a test deployment with both and it still works (both being present are the current default)